src/image-copy/azext_imagecopy/__init__.py

```diff
@@ -43,6 +43,8 @@ def load_arguments(self, _):
                        help='Name of the final image that will be created')
             c.argument('target_subscription', options_list=['--target-subscription'],
                        help='Name or ID of the subscription where the final image should be created.')
+            c.argument('time_out', options_list=['--timeout'], type=int, default=3600,
+                       help='Seconds before the command return timeout.')
```

Comment:
The help text can be more detailed to explain what is this for
```suggestion
                       help='Time in seconds for the copy operation to finish. Increase this time if you're going to copy large images (disks) like 512GB or more.')
```